### PR TITLE
Export the method to create modernUI container object

### PR DESCRIPTION
**Pull request by luckygoyal-bitmovin**
Exporting `bitmovin.playerui.UIFactory.modernUI()` method to allow application to create and use the UIContainer for a easy customization without needing to build and deploy the player UI JS.

For example hideDelay parameter can be easily customized using below code if `bitmovin.playerui.UIFactory.modernUI()` is exported as public. So if there is no side effect of making it public, it will allow for easy customizations for some cases like below.

  ```
var modernUI = new bitmovin.playerui.UIFactory.modernUI();
  modernUI.config.hideDelay = 5000;
  var modernSmallScreenUI = new bitmovin.playerui.UIFactory.modernSmallScreenUI();
  modernSmallScreenUI.config.hideDelay = 10000;

  var mainUIManager = new bitmovin.playerui.UIManager(player, [{
      ui: modernSmallScreenUI,
      condition: (context) => {
        return !context.isAd && !context.adRequiresUi && context.isMobile
          && context.documentWidth < smallScreenSwitchWidth;
      },
    }, {
      ui: modernUI,
      condition: (context) => {
        return !context.isAd && !context.adRequiresUi;
      },
    }], config);
```

### CODE REVIEW

**Review by bitmovin-kenny — APPROVED** [review]
The hideDelay should probably be adjustable via the config, but since this is the only builder method not exported 👍

**Review by stonko1994 — COMMENTED** [review]
Please add a changelog entry.

Fine for me to expose the method but please do not use it as you described in the PR description.

This is only possible in JS but per design the config is **not** modifyable after passing to a component.
```js
modernSmallScreenUI.config.hideDelay = 10000;
```

As discussed offline the whole UI structure should be created from outside and a different config should be passed to the component.

**Comment by luckygoyal-bitmovin** [comment]
> Please add a changelog entry.
> 
> Fine for me to expose the method but please do not use it as you described in the PR description.
> 
> This is only possible in JS but per design the config is **not** modifyable after passing to a component.
> 
> ```js
> modernSmallScreenUI.config.hideDelay = 10000;
> ```
> 
> As discussed offline the whole UI structure should be created from outside and a different config should be passed to the component.

Updated CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -690,6 +695,10 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
+[3.21.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.20.0...v3.11.0
```
Comment (hanneslinder) [inline CHANGELOG.md:698]: Shouldn't that be for `v3.21.0` instead?

```suggestion
[3.21.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.20.0...v3.21.0
```

Comment (luckygoyal-bitmovin) [reply CHANGELOG.md]: Thanks for catching this. Updated.

**Review by stonko1994 — COMMENTED** [review]
Please merge dev into this branch first to resolve all the release changes.

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (luckygoyal-bitmovin) [reply CHANGELOG.md]: Thanks. Done

**Review by stonko1994 — APPROVED** [review]

**Review by hanneslinder — APPROVED** [review]

**Review by offnertho — APPROVED** [review]

**Pull request merged**